<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1024</width>
                <height>768</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>标签打印编辑器</string>
        </property>
        <property name="windowIcon">
            <iconset resource="../resources/resources.qrc">
                <normaloff>:/icons/app.png</normaloff>:/icons/app.png</iconset>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <property name="leftMargin">
                    <number>0</number>
                </property>
                <property name="topMargin">
                    <number>0</number>
                </property>
                <property name="rightMargin">
                    <number>0</number>
                </property>
                <property name="bottomMargin">
                    <number>0</number>
                </property>
                <item>
                    <widget class="QSplitter" name="splitter">
                        <property name="orientation">
                            <enum>Qt::Horizontal</enum>
                        </property>
                        <widget class="LabelEditView" name="labelEditView"/>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1024</width>
                    <height>22</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>文件</string>
                </property>
                <addaction name="actionNew"/>
                <addaction name="actionOpen"/>
                <addaction name="separator"/>
                <addaction name="actionSave"/>
                <addaction name="actionSaveAs"/>
                <addaction name="separator"/>
                <addaction name="actionExportPDF"/>
                <addaction name="separator"/>
                <addaction name="actionPrint"/>
                <addaction name="actionPrintPreview"/>
                <addaction name="separator"/>
                <addaction name="actionExit"/>
            </widget>
            <widget class="QMenu" name="menuEdit">
                <property name="title">
                    <string>编辑</string>
                </property>
                <addaction name="actionUndo"/>
                <addaction name="actionRedo"/>
                <addaction name="separator"/>
                <addaction name="actionCut"/>
                <addaction name="actionCopy"/>
                <addaction name="actionPaste"/>
                <addaction name="actionDelete"/>
                <addaction name="separator"/>
                <addaction name="actionSelectAll"/>
                <addaction name="actionDeselectAll"/>
            </widget>
            <widget class="QMenu" name="menuInsert">
                <property name="title">
                    <string>插入</string>
                </property>
                <addaction name="actionAddText"/>
                <addaction name="actionAddImage"/>
                <addaction name="actionAddBarcode"/>
                <addaction name="actionAddQRCode"/>
            </widget>
            <widget class="QMenu" name="menuView">
                <property name="title">
                    <string>视图</string>
                </property>
                <addaction name="actionZoomIn"/>
                <addaction name="actionZoomOut"/>
                <addaction name="actionZoomReset"/>
                <addaction name="actionZoomFit"/>
                <addaction name="separator"/>
                <addaction name="actionShowGrid"/>
                <addaction name="actionShowRulers"/>
                <addaction name="actionSnapToGrid"/>
            </widget>
            <widget class="QMenu" name="menuSettings">
                <property name="title">
                    <string>设置</string>
                </property>
                <addaction name="actionPageSetup"/>
                <addaction name="actionPrinterSettings"/>
            </widget>
            <widget class="QMenu" name="menuHelp">
                <property name="title">
                    <string>帮助</string>
                </property>
                <addaction name="actionAbout"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuEdit"/>
            <addaction name="menuInsert"/>
            <addaction name="menuView"/>
            <addaction name="menuSettings"/>
            <addaction name="menuHelp"/>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <widget class="QToolBar" name="toolBarFile">
            <property name="windowTitle">
                <string>文件工具栏</string>
            </property>
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <addaction name="actionNew"/>
            <addaction name="actionOpen"/>
            <addaction name="actionSave"/>
            <addaction name="actionPrint"/>
        </widget>
        <widget class="QToolBar" name="toolBarEdit">
            <property name="windowTitle">
                <string>编辑工具栏</string>
            </property>
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <addaction name="actionUndo"/>
            <addaction name="actionRedo"/>
            <addaction name="separator"/>
            <addaction name="actionCut"/>
            <addaction name="actionCopy"/>
            <addaction name="actionPaste"/>
            <addaction name="actionDelete"/>
        </widget>
        <widget class="QToolBar" name="toolBarInsert">
            <property name="windowTitle">
                <string>插入工具栏</string>
            </property>
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <addaction name="actionAddText"/>
            <addaction name="actionAddImage"/>
            <addaction name="actionAddBarcode"/>
            <addaction name="actionAddQRCode"/>
        </widget>
        <widget class="QToolBar" name="toolBarView">
            <property name="windowTitle">
                <string>视图工具栏</string>
            </property>
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <addaction name="actionZoomIn"/>
            <addaction name="actionZoomOut"/>
            <addaction name="actionZoomReset"/>
            <addaction name="actionZoomFit"/>
        </widget>
        <action name="actionNew">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
            </property>
            <property name="text">
                <string>新建</string>
            </property>
            <property name="toolTip">
                <string>创建新文档</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+N</string>
            </property>
        </action>
        <action name="actionOpen">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
            </property>
            <property name="text">
                <string>打开...</string>
            </property>
            <property name="toolTip">
                <string>打开现有文档</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+O</string>
            </property>
        </action>
        <action name="actionSave">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
            </property>
            <property name="text">
                <string>保存</string>
            </property>
            <property name="toolTip">
                <string>保存文档</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+S</string>
            </property>
        </action>
        <action name="actionSaveAs">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/save-as.png</normaloff>:/icons/save-as.png</iconset>
            </property>
            <property name="text">
                <string>另存为...</string>
            </property>
            <property name="toolTip">
                <string>以新名称保存文档</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Shift+S</string>
            </property>
        </action>
        <action name="actionExportPDF">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/export.png</normaloff>:/icons/export.png</iconset>
            </property>
            <property name="text">
                <string>导出为PDF...</string>
            </property>
            <property name="toolTip">
                <string>将文档导出为PDF</string>
            </property>
        </action>
        <action name="actionPrint">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/print.png</normaloff>:/icons/print.png</iconset>
            </property>
            <property name="text">
                <string>打印...</string>
            </property>
            <property name="toolTip">
                <string>打印文档</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+P</string>
            </property>
        </action>
        <action name="actionPrintPreview">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/print-preview.png</normaloff>:/icons/print-preview.png</iconset>
            </property>
            <property name="text">
                <string>打印预览...</string>
            </property>
            <property name="toolTip">
                <string>预览打印效果</string>
            </property>
        </action>
        <action name="actionExit">
            <property name="text">
                <string>退出</string>
            </property>
            <property name="toolTip">
                <string>退出应用程序</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Q</string>
            </property>
        </action>
        <action name="actionUndo">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
            </property>
            <property name="text">
                <string>撤销</string>
            </property>
            <property name="toolTip">
                <string>撤销上一操作</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Z</string>
            </property>
        </action>
        <action name="actionRedo">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
            </property>
            <property name="text">
                <string>重做</string>
            </property>
            <property name="toolTip">
                <string>重做上一操作</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Y</string>
            </property>
        </action>
        <action name="actionCut">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/cut.png</normaloff>:/icons/cut.png</iconset>
            </property>
            <property name="text">
                <string>剪切</string>
            </property>
            <property name="toolTip">
                <string>剪切选中的元素</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+X</string>
            </property>
        </action>
        <action name="actionCopy">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
            </property>
            <property name="text">
                <string>复制</string>
            </property>
            <property name="toolTip">
                <string>复制选中的元素</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+C</string>
            </property>
        </action>
        <action name="actionPaste">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/paste.png</normaloff>:/icons/paste.png</iconset>
            </property>
            <property name="text">
                <string>粘贴</string>
            </property>
            <property name="toolTip">
                <string>粘贴剪贴板内容</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+V</string>
            </property>
        </action>
        <action name="actionDelete">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
            </property>
            <property name="text">
                <string>删除</string>
            </property>
            <property name="toolTip">
                <string>删除选中的元素</string>
            </property>
            <property name="shortcut">
                <string>Del</string>
            </property>
        </action>
        <action name="actionSelectAll">
            <property name="text">
                <string>全选</string>
            </property>
            <property name="toolTip">
                <string>选择所有元素</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+A</string>
            </property>
        </action>
        <action name="actionDeselectAll">
            <property name="text">
                <string>取消选择</string>
            </property>
            <property name="toolTip">
                <string>取消选择所有元素</string>
            </property>
            <property name="shortcut">
                <string>Esc</string>
            </property>
        </action>
        <action name="actionAddText">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/text.png</normaloff>:/icons/text.png</iconset>
            </property>
            <property name="text">
                <string>添加文本</string>
            </property>
            <property name="toolTip">
                <string>添加文本元素</string>
            </property>
        </action>
        <action name="actionAddImage">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/image.png</normaloff>:/icons/image.png</iconset>
            </property>
            <property name="text">
                <string>添加图像</string>
            </property>
            <property name="toolTip">
                <string>添加图像元素</string>
            </property>
        </action>
        <action name="actionAddBarcode">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/barcode.png</normaloff>:/icons/barcode.png</iconset>
            </property>
            <property name="text">
                <string>添加条形码</string>
            </property>
            <property name="toolTip">
                <string>添加条形码元素</string>
            </property>
        </action>
        <action name="actionAddQRCode">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/qrcode.png</normaloff>:/icons/qrcode.png</iconset>
            </property>
            <property name="text">
                <string>添加二维码</string>
            </property>
            <property name="toolTip">
                <string>添加二维码元素</string>
            </property>
        </action>
        <action name="actionZoomIn">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
            </property>
            <property name="text">
                <string>放大</string>
            </property>
            <property name="toolTip">
                <string>放大视图</string>
            </property>
            <property name="shortcut">
                <string>Ctrl++</string>
            </property>
        </action>
        <action name="actionZoomOut">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
            </property>
            <property name="text">
                <string>缩小</string>
            </property>
            <property name="toolTip">
                <string>缩小视图</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+-</string>
            </property>
        </action>
        <action name="actionZoomReset">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/zoom-reset.png</normaloff>:/icons/zoom-reset.png</iconset>
            </property>
            <property name="text">
                <string>重置缩放</string>
            </property>
            <property name="toolTip">
                <string>重置为100%缩放</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+0</string>
            </property>
        </action>
        <action name="actionZoomFit">
            <property name="icon">
                <iconset resource="../resources/resources.qrc">
                    <normaloff>:/icons/zoom-fit.png</normaloff>:/icons/zoom-fit.png</iconset>
            </property>
            <property name="text">
                <string>适应窗口</string>
            </property>
            <property name="toolTip">
                <string>缩放到适应窗口大小</string>
            </property>
        </action>
        <action name="actionShowGrid">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="checked">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>显示网格</string>
            </property>
            <property name="toolTip">
                <string>显示或隐藏网格</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+G</string>
            </property>
        </action>
        <action name="actionShowRulers">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="checked">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>显示标尺</string>
            </property>
            <property name="toolTip">
                <string>显示或隐藏标尺</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+R</string>
            </property>
        </action>
        <action name="actionSnapToGrid">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="checked">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>对齐到网格</string>
            </property>
            <property name="toolTip">
                <string>启用或禁用网格对齐</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Shift+G</string>
            </property>
        </action>
        <action name="actionPageSetup">
            <property name="text">
                <string>页面设置...</string>
            </property>
            <property name="toolTip">
                <string>配置页面大小和方向</string>
            </property>
        </action>
        <action name="actionPrinterSettings">
            <property name="text">
                <string>打印机设置...</string>
            </property>
            <property name="toolTip">
                <string>配置打印机选项</string>
            </property>
        </action>
        <action name="actionAbout">
            <property name="text">
                <string>关于</string>
            </property>
            <property name="toolTip">
                <string>显示应用程序信息</string>
            </property>
        </action>
    </widget>
    <customwidgets>
        <customwidget>
            <class>LabelEditView</class>
            <extends>QGraphicsView</extends>
            <header>ui/labeleditview.h</header>
        </customwidget>
    </customwidgets>
    <resources>
        <include location="../resources/resources.qrc"/>
    </resources>
    <connections/>
</ui>